llama_stack/cli/stack/build.py

```diff
@@ -296,6 +292,7 @@ def _run_stack_build_command_from_build_config(
                 / f"templates/{template_name}/run.yaml"
             )
             with importlib.resources.as_file(template_path) as path:
+                run_config_file = build_dir / f"{build_config.name}-run.yaml"
```

Comment:
cc @terrytangyuan this was a bug from one of your stack build related PRs, just FYI